common/config/rush/.npmrc

```diff
@@ -1 +0,0 @@
-registry=https://registry.npmjs.org/
```

Comment:
We might still want the equivalent of this to prevent weirdness from being introduced by people who work on internal repos where they globally set the registry to a private one.